dask/array/tests/test_ghost.py

```diff
@@ -1,6 +1,9 @@
-import dask.array as da
+import pytest
```

Comment:
No real concern here.  I would keep this.  Something to be aware of in the future though.